src/components/AppNavigation/CalendarList/CalendarListItem.vue

```diff
@@ -31,9 +31,15 @@
 				<ActionButton @click.prevent.stop="toggleEnabled">
 					<template #icon>
 						<CheckboxBlankCircle
-							:title="calendar.enabled ? $t('calendar', 'Disable calendar') : $t('calendar', 'Enable calendar') "
+							v-if="calendar.enabled"
+							:title="$t('calendar', 'Disable calendar')"
 							:size="20"
-							:fill-color="calendar.enabled ? calendar.color : 'var(--color-text-lighter)'" />
+							:fill-color="calendar.color" />
+						<CheckboxBlankCircleOutline
+							v-else
+							:title="$t('calendar', 'Enable calendar')"
+							:size="20"
+							fill-color="var(--color-text-lighter)" />
```

Comment:
@jancborchardt This would set the color to the calendars color.
```suggestion
							:fill-color="calendar.color" />
```